WordPress/Classes/Utility/ContextManager.m

```diff
@@ -350,6 +350,9 @@ - (void)migrateDataModelsIfNecessary
 
         if (error != nil) {
             DDLogError(@"Unable to migrate store: %@", error);
+
+            SentryStartupEvent *startupEvent = [SentryStartupEvent new];
```

Comment:
We need to use the same `startupEvent` from `persistentContainer`. Otherwise we will end up with different events and this one will not be triggered (we need to call `sendWithTitle:` to fire this).

Take a look here: https://github.com/wordpress-mobile/WordPress-iOS/blob/fef99452a26bf739fa997e711d1f0817f57c9705/WordPress/Classes/Utility/ContextManager.m#L252